pkg/reconciler/testing/v1/eventtype.go

```diff
@@ -23,16 +23,16 @@ import (
 	duckv1 "knative.dev/pkg/apis/duck/v1"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"knative.dev/eventing/pkg/apis/eventing/v1beta2"
+	"knative.dev/eventing/pkg/apis/eventing/v1beta3"
 	"knative.dev/pkg/apis"
 )
 
 // EventTypeOption enables further configuration of an EventType.
-type EventTypeOption func(*v1beta2.EventType)
+type EventTypeOption func(*v1beta3.EventType)
 
 // NewEventType creates a EventType with EventTypeOptions.
-func NewEventType(name, namespace string, o ...EventTypeOption) *v1beta2.EventType {
-	et := &v1beta2.EventType{
+func NewEventType(name, namespace string, o ...EventTypeOption) *v1beta3.EventType {
+	et := &v1beta3.EventType{
```

Comment:
I'm happy to update this either in this PR or a separate PR if we want, but this file seemed to be using `v1beta2` so I had assumed it would be fine to bump to `v1beta3` :)